src/main/java/com/google/api/codegen/discogapic/transformer/java/JavaDiscoGapicSchemaToViewTransformer.java

```diff
@@ -105,22 +108,32 @@ public int compare(Schema o1, Schema o2) {
 
   private SchemaTypeTable createTypeTable(String implicitPackageName) {
     return new SchemaTypeTable(
-        new JavaTypeTable(implicitPackageName),
-        new JavaSchemaTypeNameConverter(implicitPackageName));
+        new JavaTypeTable(implicitPackageName, true),
```

Comment:
booleans can be hard to read as parameters, since you can't tell at the call site what true/false means. I would suggest using an enum.